src/metabase/driver/util.clj

```diff
@@ -316,6 +316,22 @@
                  db-details
                  secret-names->props))))
 
+(def official-drivers
+  "List of all official drivers"
+  ["bigquery-cloud-sdk" "druid" "googleanalytics" "h2" "mongo" "mysql" "postgres" "presto" "presto-jdbc" "redshift" "snowflake" "sparksql" "sqlite" "sqlserver"])
+
+(def partner-drivers
+  "List of other drivers in the partnership program"
+  ["firebolt"])
+
+(defn driver-source
+  "Return the source type of the driver: official, partner, or community"
+  [driver-name]
+  (cond
+    (contains? (set official-drivers) driver-name) "official"
```

Comment:
It would be even better if we could put this info in the driver hierarchy somehow so you could instead just do `(isa? driver/hierarchy my-driver ::driver/community-driver)` or something like that but I guess that could be tricky if someone does a community driver deriving from an official driver

Reply:
this would let any 3rd party driver report as a first party though. I don't see too many downsides to a list of drivers in the source code. The list of first party drivers is very stable and certainly would be noticed in QA before releasing. If we fail to notice it though, the only issue would be that it would incorrectly show a banner that it is a third party driver. I'm in favor or the simpler solutions here rather than more complicated ones.

Reply:
The drivers are now sets instead of lists.